### PR TITLE
WIP- add pact test

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1278

### Change description ###

add pact test

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationContractTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @LocalServerPort
+    private int port;
+
+    @MockBean
+    private AuthService authService;
+
+    @BeforeEach
+    void setupTestTarget(PactVerificationContext context) {
+        context.setTarget(new HttpTestTarget("localhost", port, "/"));
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationContractTest.java:36]: will this work in pipeline ?

Comment (mdayican) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationContractTest.java:36]: we will see :)

**Comment by rkondratowicz** [comment]
looks like there are missing changes in jenkinsfile

File: build.gradle [hunk 2]
```diff
@@ -190,6 +190,7 @@ dependencies {
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
+  testImplementation group: 'au.com.dius', name: 'pact-jvm-provider-junit5_2.12', version: '3.6.15'
```
Comment (rkondratowicz) [inline build.gradle:193]: 4.0.10 is available

**Pull request closed without merging**